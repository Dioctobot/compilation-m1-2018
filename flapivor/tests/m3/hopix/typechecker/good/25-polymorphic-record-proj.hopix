type P<a, b> = { x : a; y : b }

val p : P<Int, String> = { x = 0; y = "Rose Tyler" }<Int, String>

val x : Int = p.x

val y : String = p.y