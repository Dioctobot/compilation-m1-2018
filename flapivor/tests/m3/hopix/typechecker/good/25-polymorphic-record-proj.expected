p : P<Int, String>
x : Int
y : String
